### PR TITLE
remove xpub for pushes

### PR DESCRIPTION
**Pull request by Pantani**
remove xpub implementation for push notifications

closes https://github.com/trustwallet/blockatlas/issues/705

### CODE REVIEW

**Comment by Pantani** [comment]
> Can you add test cases: [#705 (comment)](https://github.com/trustwallet/blockatlas/issues/705#issuecomment-574445599) in your tests to make sure it send correct push notification webbook

What type of test do you mean? integration tests? because all of the testable methods already have tested, the other methods need database connection and do request to the push services

**Comment by vikmeup** [comment]
unit test, are you sure you have test for change address to yourself? just double cheking

File: observer/observer.go [hunk 1]
```diff
@@ -133,3 +94,15 @@ func getDirection(tx blockatlas.Tx, address string) blockatlas.Direction {
 	}
 	return blockatlas.DirectionOutgoing
 }
+
+func inferUtxoValue(tx *blockatlas.Tx, address string, coinIndex uint) {
+	if len(tx.Inputs) > 0 && len(tx.Outputs) > 0 {
```
Comment (vikmeup) [inline observer/observer.go]: maybe return an error here?

Comment (Pantani) [reply observer/observer.go]: no, because if is not an utxo tx, the method don't going do to anything

Comment (vikmeup) [reply observer/observer.go]: Maybe add property to transaction as type, and only call this function if it’s utxo transaction type?

**Pull request merged**